query/parser.go

```diff
@@ -340,8 +384,12 @@ func (p *Parser) insertPropertyKeyValue() {
 	case "from":
 		fallthrough
 	case "to":
-		unix, err := parseDate(value)
-		if err != nil {
+		var unix int64
+		var err error
+		if unix, err = parseDate(value); err == nil {
+			// Valid, so do nothing.
+		} else if unix, err = parseRelativeTime(value, time.Now().UTC()); err != nil {
```

Comment:
In practice, probably not a big deal. There might be a difference of 1 second between what `from` and `to` think `now` is.

For replayability, just log the timestamps when queries run or don't use the feature. It's syntactic sugar for running commands on the fly.